app/templates/main/request-received.html

```diff
@@ -12,15 +12,15 @@ <h2 class="govuk-heading-m govuk-!-margin-top-7">
             What happens next
         </h2>
         <p class="govuk-body">
-            We will email a link to <span class="govuk-!-font-weight-bold">{{ context["user_email"] }}</span>.
+            We will email a link to <span class="govuk-!-font-weight-bold">{{ user_email }}</span>.
         </p>
         <p class="govuk-body">
             This may take up to 5 minutes to be delivered to your inbox.</p>
         <h2 class="govuk-heading-m">
-            If you’ve not received the email</h2>
+            If you've not received the email</h2>
```

Comment:
Have you changed these apostrophes manually or were they caught by some auto-formatter? They were originally 'fancy' quotes and I find that design-minded people generally prefer them over simple ASCII apostrophes :)

Reply:
I changed them, because I always had these when I copied some text from a word document, and then sometimes became weird symbols on rendered html page. Changed them back now to the original quotes.